code/script.js

```diff
@@ -1,47 +1,282 @@
+    botReply('You have to type either "stocks" or "funds".');
+  }
+};
+
+/* This function will print a speech bubble on the screen with the users answer which should be either "high" or "low". It will then check whcih type of stock the user want. Then push the user in the right direction.*/
+const highRiskOrLowRiskStockQuestion = (message) => {
+  questionNumber = 99;
+  userReply(message); // Speechbubble high
+  formMessageBox.value = "";
+  if (message === "high" || message === "high risk") {
+    botReply("oh, so you want high risk!");
+    setTimeout(chatDelay);
+    botReply("Here you go!");
+
+    /*At this poin in the code we have coded a HTML slect option list that will be rendered on the screen and our previous textbox will be removed*/
+    formInputWrapper.innerHTML = `
+  <div class="input-wrapper" id="input-wrapper">
+    <div class="selectOptionDiv" id="selectOptionDiv">
+      <select id="selectMenuOptions">
+        <option value="" selected disabled>Select a stock</option>
+        <option value="Provide IT">Provide IT</option>
+        <option value="Sagax D">Sagax D</option>
+        <option value="Naxs">Naxs</option>
+        <option value="Lundin Energy">Lundin Energy</option>
+      </select>
+      <button id="selectOptionButton" class="send-btn" type="button">Buy</button>
+    </div>
+  </div>`;
+
+    let selectOptionButton = document.getElementById("selectOptionButton");
+    let selectMenuOptions = document.getElementById("selectMenuOptions");
+
+    /* We had to add a new event lsitener to our "buy" button we rendered above. */
+    selectOptionButton.addEventListener("click", () =>
+      nextQuestion(selectMenuOptions.value)
+    );
+  } else if (message === "low" || message === "low risk") {
+    botReply("You want low risk");
+    setTimeout(chatDelay);
+    botReply("Here you go!");
+
+    /* At this poin in the code we have coded a HTML slect option list that will be rendered on the screen and our previous textbox will be removed */
+    formInputWrapper.innerHTML = `
+    <div class="input-wrapper" id="input-wrapper">
+    <div class="selectOptionDiv" id="selectOptionDiv">
+    <select id="selectMenuOptions">
+    <option value="" selected disabled>Select a stock</option>
+    <option value="Investor">Investor</option>
+    <option value="Latour">Latour</option>
+    <option value="Kinnevik">Kinnevik</option>
+    <option value="Castellum">Castellum</option>
+    </select>
+    <button id="selectOptionButton" class="send-btn" type="button">Buy</button>
+    </div>
+    </div>`;
+    let selectOptionButton = document.getElementById("selectOptionButton");
+    let selectMenuOptions = document.getElementById("selectMenuOptions");
+
+    /* We had to add a new event lsitener to our "buy" button we rendered above. */
+    selectOptionButton.addEventListener("click", () =>
+      nextQuestion(selectMenuOptions.value)
+    );
+  } else {
+    botReply("You have to type either low risk or high risk.");
+  }
+};
+
```

Comment:
You created 2 very long and very similar functions. I think it is a better way to create a more generic "template" function, and pass data to it via parameters like it is done in `showMessage` function 🤓